awx/main/conf.py

```diff
@@ -694,16 +694,18 @@
     category_slug='logging',
 )
 register(
-    'LOG_AGGREGATOR_MAX_DISK_USAGE_GB',
```

Comment:
I wonder what happens with this setting in the DB? I guess there isn't much harm in letting it continue to live in the DB. 

I'm imagining the developer that comes along down the road and picks this same setting name for a different purpose & is confused by the bug reports that come with it. Slim chances, but a funny thought.